Dockerfile

```diff
@@ -13,10 +13,11 @@ ARG DOCKER_IMAGE
 RUN apt-get update && apt-get install build-essential -y
 
 # Pod tasks should be exposed in the default image
-RUN pip install -U flytekit==$VERSION \
+RUN pip install --no-cache-dir -U flytekit==$VERSION \
 	flytekitplugins-pod==$VERSION \
 	flytekitplugins-deck-standard==$VERSION \
-	scikit-learn
+	scikit-learn \
+	&& :
```

Comment:
Why is this necessary?

Reply:
Great catch should not be needed

Reply:
Adding `&& :` is a no-op. This helps in maintaining git history (if we want to remove `scikit-learn` or install another package you don't have to add another symbol to that line).

Reply:
Ah cool!